cmake/modules/OpenSSL.cmake

```diff
@@ -10,6 +10,7 @@
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 #
+mark_as_advanced(OPENSSL_BINARY)
```

Comment:
It seems to me this is also in #1496, Angelo

Do you want to mark variables as advanced in this PR or in that one?

Because, depending on what you want to do (two separate PRs, or all in one), we can arrange PRs accordingly :)

Reply:
It's here by mistake, I intended it to be in the other PR.